### PR TITLE
Add index.html

### PR DESCRIPTION
**Pull request by me-jterhorst**
Hey this is my personal website, have a look and let me know what you think!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Jakob ter Horst | Personal Website</title>
+  </head>
+  <body>
+    <header class="top">
+      <div class="top__logo">
+        <img src="" alt="Logo" class="top__logo__img" />
+      </div>
```
Comment (mpagels) [inline index.html:13]: Do you need the div here?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      <nav>
+        <ul class="top__nav">
```
Comment (mpagels) [inline index.html:15]: BEM style. I like.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+          <li><a href="#stack">Tech Stack</a></li>
+          <li><a href="#cv">Links of Interest</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section id="about">
+        <div class="about__hero">
+          <img class="about__hero__img" src="" alt="" />
```
Comment (mpagels) [inline index.html:27]: Here is the source to the image missing.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+            commodi! Et excepturi accusamus dicta assumenda mollitia nostrum
+            tempora ullam, distinctio voluptatum optio maiores modi deserunt.
+            Nulla, numquam quasi!
+          </p>
+        </div>
+      </section>
+
+      <section id="portfolio">
+        <div class="portfolio__card">
+          <img class="portfolio__card__img" src="" alt="Hero" />
```
Comment (mpagels) [inline index.html:53]: see line 27

File: index.html [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+            <li>Projekt 3</li>
+            <li>Projekt 4</li>
+            <li>Projekt 5</li>
+          </ol>
+        </div>
+      </section>
+
+      <section id="stack">
+        <div class="stack__card">
+          <img class="stack__card__img" src="" alt="" />
```
Comment (mpagels) [inline index.html:69]: see line 27

File: index.html [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+        <div>
+          <button>Lebenslauf</button>
+        </div>
+
+        <div>
+          <button>Zertifikate</button>
+        </div>
+        <div>
+          <button>Anschreiben</button>
+        </div>
```
Comment (mpagels) [inline index.html:93]: Do you need the divs?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+      </aside>
+    </main>
+    <footer>
+      <ul>
+        <li><a href="#">Datenschutz</a></li>
+        <li><a href="#">Impressum</a></li>
+        <li>Copyright: 17.06.2021</li>
+        <li>Jakob ter Horst</li>
+      </ul>
+    </footer>
```
Comment (mpagels) [inline index.html:103]: I like that.

**Review by mpagels — CHANGES_REQUESTED** [review]
Hey Jakob. Very good job here.
I found some missing sources for the images. Pls check.
some `div` could be avoided I think.

Very good that you are using BEM naming for the css classes.

File: index.html (hunk 1 above)
Comment (me-jterhorst) [reply index.html:13]: Thank you for your feedback :) 
My reasoning is that I'm going to give the img element an object-fit property in css later to add a more responsive feel.

File: index.html (hunk 6 above)
Comment (me-jterhorst) [reply index.html:93]: I thought about adding additional icons just before the buttons, 
that's why I wrapped them in divs already. 
I am unsure wether or not to use emojis as p or actual png/svg icons

File: index.html (hunk 1 above)
Comment (mpagels) [reply index.html:13]: A pull request into `main` is somehting like -> I put something into `production`. If you want something add later, it should not be part of the pull request. Think of it like a car manufactor put a car update in production and say to you "I will put the breaks later into the car" You don't want that. ;-)

File: index.html (hunk 6 above)
Comment (mpagels) [reply index.html:93]: You can use emojis or svg icons. That is up to you. But is a bit like this example here:
https://github.com/me-jterhorst/my-personal-website/pull/1#discussion_r653690645

**Pull request closed without merging**